src/components/events/Event/index.js

```diff
@@ -23,16 +22,25 @@ module.exports = React.createClass({
     };
   },
 
-  // componentWillMount: function() {
-  //   var _this = this;
-  //   var backgroundImage = document.createElement('img');
-  //   backgroundImage.src = this.props.backgroundImageUrl;
-  //   backgroundImage.onload = function() {
-  //     _this.setState({
-  //       blurredImage: effect.blurImage(backgroundImage, 30)
-  //     });
-  //   };
-  // },
+  // () -> ()
+  getInitialState: function() {
+    return {
+      blurredBackgroundImage: ''
+    };
+  },
+
+  // () -> ()
+  componentWillMount: function() {
+    var _this = this;
+    var backgroundImage = document.createElement('img');
+    backgroundImage.setAttribute('crossorigin', 'anonymous');
+    backgroundImage.src = this.props.backgroundImageUrl;
+    backgroundImage.onload = function() {
+      _this.setState({
+        blurredBackgroundImage: effect.blurImage(backgroundImage, 30)
+      });
+    };
+  },
 
   // () -> XJS
```

Comment:
Sorry @ryanbooker. Didn't mean to pile all this on you. Specially since your new to the React world.

Just trying to keep the worlds worst language as tidy as we can. We've found commenting is a slippery slope to random, out-off-date explanations of code that is usually clearer on it's own.

If you feel strongly about commenting, then please bring it up at the next JS guild meeting or you can put a PR up to the JS style guild to spark a discussion. 

If you don't feel strongly about it, if you remove the comments I'll merge your code. ;)